tests/manage/pv_services/test_pvc_resize.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import skipif_ocs_version
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version(['>4.1', '<4.5'])
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-00000")
+        ),
+        pytest.param(
+            constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-0000")
+        )
+    ]
+)
+def test_pvc_resize(interface, storageclass_factory, pvc_factory, pod_factory):
+    """
+    Verify PVC resize feature
+    """
+    pvc_size = 10
+    pvc_size_new = 40
+    patch_param = f'{{"spec": {{"resources": {{"requests": {{"storage": "{pvc_size_new}Gi"}}}}}}}}'
```

Comment:
could you please used ocs_ci.ocs.resources.pvc.PVC#resize_pvc.
It was created a long time ago so probably needs to fix

Reply:
Done.